packages/languages/es/src/index.ts

```diff
@@ -0,0 +1,15 @@
+import commonWords from './commonWords.json'
+import firstnames from './firstnames.json'
+import lastnames from './lastnames.json'
+import wikipedia from './wikipedia.json'
+import translations from './translations'
+
+export default {
+  dictionary: {
+    commonWords,
+    firstnames,
+    lastnames,
+    wikipedia,
```

Comment:
Either add an empty wikipedia.json file, add the wikipedia data with following the documentation secondNavigation or remove it.
If you leave it empty or remove it i will add it afterwards

Reply:
Added an empty wikipedia.json. I hope you can process it on your side as I'm currently on mobile data. If I download the wikipedia data it will make a huge dent on my limits :)